backend/app/legacy/controllers/info.py

```diff
@@ -82,3 +93,37 @@ def healthcheck() -> (Healthcheck, int):
         return healthcheck_status, 500
 
     return healthcheck_status, 200
+
+
+def sync_status() -> Tuple[SyncStatus, int]:
```

Comment:
controller should only be a facade for the logic that'll be put into the service; so the logic from here should be moved to the core/ package